buildenv/jenkins/JenkinsfileBase

```diff
@@ -67,8 +67,10 @@ def test() {
 			timestamps{
 				echo 'Building tests...'
 				sh "chmod 755 ${JAVA_BIN}/java"
-				sh "chmod 755 ${JAVA_BIN}/../../bin/javac"
-				sh "chmod 755 ${JAVA_BIN}/../../bin/java"
+				if (JAVA_VERSION == 'SE80') {
+					sh "chmod 755 ${JAVA_BIN}/../../bin/javac"
+					sh "chmod 755 ${JAVA_BIN}/../../bin/java"
+				}
```

Comment:
what if it's not SE80?

What's the difference between 
```sh "chmod 755 ${JAVA_BIN}/java"
sh "chmod 755 ${JAVA_BIN}/../../bin/java
```

Reply:
When we are on SE80, 

`${JAVA_BIN}/java`=`$WORKSPACE/openjdkbinary/j2sdk-image/jre/bin/java` 
`${JAVA_BIN}/../../bin/java`=`$WORKSPACE/openjdkbinary/j2sdk-image/bin/java` 

While running on the internal IBM Java 8 SR6, I discovered that there are some tests that use the `"$WORKSPACE/openjdkbinary/j2sdk-image/bin/java` and `"$WORKSPACE/openjdkbinary/j2sdk-image/bin/javac`. That's why chmod'ing those executables, when we are on SE80, is required. 

The problem was only seen in Java 8 SR6 IBM internal builds, so, I will remove the chmod step which is outside of the if condition.